release-note-script/README.md

```diff
@@ -0,0 +1,60 @@
+# Release Note Script
+
+These are scripts for generating the main body of release notes for Scalar products. There are two scripts available:
+
+- ReleaseNoteCreation.java ... Creates a release note body for a target repository.
+- MergeReleaseNotes.java ... Creates a merged release note body for ScalarDB (including community and enterprise edition).
+
+These scripts are invoked in GitHub Actions workflows when releasing a new version of Scalar products. 
+
+For the details of the release note please see [General rules]().
+
+## Prerequisites
+
+- [GitHub CLI](https://cli.github.com/)
+- Java 11+
+
+## How to Run the Scripts
+
+Note that primarily, these scripts are intended to be executed within GitHub Actions workflows.
+So usually you don't need to execute the scripts manually.
+
+### Create a Release Note Body for Each Repository
+
+*Assuming the use of Java 11*
+To run the script:
+
+```shell
+java --source 11 ReleaseNoteCreation.java <owner> <projectTitlePrefix> <version> <repository>
+```
+
+Here, the arguments mean the following:
+
+- *owner*: Owner of the target repository.
+- *projectTitlePrefix*: Prefix for the target version project.
+  - e.g., If the project name of the target is `ScalarDB 4.0.0`, then specify `ScalarDB`. 
+- *version*: The target version to create the release note.
+  - e.g., If the project name of the target is `ScalarDB 4.0.0`, then specify `4.0.0`. 
+- *repository*: The name of the target repository.
+
+Example: To create the release note for ScalarDB 4.0.0
+
+```shell
+java --source 11 ReleaseNoteCreation.java scalar-labs ScalarDB 4.0.0 scalardb
```

Comment:
Just out of curiosity, but I'm wondering why this application isn't executed via https://docs.gradle.org/current/userguide/application_plugin.html. I think you can use convenient libraries through Gradle.

Reply:
Thank you for the comment.

Certainly, that's right. However, here are the reasons why I didn't do it that way:
- I wanted to keep it simple and executable as a single file, similar to a scripting language, as it's only meant to generate release notes.
- The code size isn't big, and there aren't many complex operations, so I didn't think it was necessary to use an external library.
- I chose to use Gradle for the testing automation only.
Of course, if this script becomes more complex in the future, I intend to consider options like using external libraries with the `application` plugin.